File: sumCarryBitwise.java
package springIntro;

public class sumCarryBitwise {

	public static void main(String[] args) {
		boolean x = true;
		boolean y = false;
		boolean sum = x ^ y;
		boolean get  = x & y;
		System.out.print("X\tY\tSum\tGet\n");
		System.out.print(x + "\t" + y + "\t" + sum + "\t" + get);
	}
}

File: sumCarryBitwise.cpp
#include<iostream>
using std :: cout;
using std :: endl;
int main(){
	int x = 1, y = 0;
	int get = 0, sum = 0;
	sum = x ^ y;
	get = x & y;
	cout << "X\tY\tSum\tGet" << endl;
	cout << x << "\t" << y << "\t" << sum << "\t" << get;
	return 0;
}
